<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiscountBondDialog</class>
 <widget class="QDialog" name="DiscountBondDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>362</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="gbBondTransactionDetails">
     <property name="title">
      <string>Discount Bond Transaction Details</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblOperationType">
        <property name="text">
         <string>Operation Type :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbOperationType"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblISINCode">
        <property name="text">
         <string>ISIN Code :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leISINCode">
        <property name="maxLength">
         <number>50</number>
        </property>
        <property name="placeholderText">
         <string>TR010120185</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblTerm">
        <property name="text">
         <string>Term :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDateEdit" name="deTerm">
        <property name="displayFormat">
         <string>MMMM dd, yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblNominalValue">
        <property name="text">
         <string>Nominal Value :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="dsbNominalValue">
        <property name="showGroupSeparator" stdset="0">
         <bool>true</bool>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="maximum">
         <double>10000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblSalePrice">
        <property name="text">
         <string>Sale Price :</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="dsbSalePrice">
        <property name="showGroupSeparator" stdset="0">
         <bool>true</bool>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="maximum">
         <double>10000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lblCount">
        <property name="text">
         <string>Count:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="lblOperationDate">
        <property name="text">
         <string>Operation Date :</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QDateEdit" name="deDate">
        <property name="maximumDate">
         <date>
          <year>2250</year>
          <month>12</month>
          <day>31</day>
         </date>
        </property>
        <property name="minimumDate">
         <date>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
        <property name="displayFormat">
         <string>MMMM dd, yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="sbCount">
        <property name="maximum">
         <number>10000000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bbTransactionApproval">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vlStatusBar"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbOperationType</tabstop>
  <tabstop>leISINCode</tabstop>
  <tabstop>deTerm</tabstop>
  <tabstop>dsbNominalValue</tabstop>
  <tabstop>dsbSalePrice</tabstop>
  <tabstop>sbCount</tabstop>
  <tabstop>deDate</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>bbTransactionApproval</sender>
   <signal>accepted()</signal>
   <receiver>DiscountBondDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bbTransactionApproval</sender>
   <signal>rejected()</signal>
   <receiver>DiscountBondDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
